focus-ios/focus-ios-tests/XCUITest/SettingAppearanceTest.swift

```diff
@@ -106,43 +107,44 @@ class SettingAppearanceTest: BaseTestCase {
         reviewCell.tap()
         waitForExistence(safariApp, timeout: 10)
         XCTAssert(safariApp.state == .runningForeground)
-        // Commenting this part due to error re-launching the app
-        /* safariApp.terminate()
+
+        safariApp.terminate()
         XCUIDevice.shared.press(.home)
+
         // Let's be sure the app is backgrounded
         let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
         waitForExistence(springboard.icons["XCUITest-Runner"], timeout: 15)
         app.activate()
-        waitForExistence(app.navigationBars["Settings"], timeout: 10)*/
+        waitForExistence(app.navigationBars["Settings"], timeout: 10)
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2574974
     func testOpenInSafari() {
         let safariapp = XCUIApplication(privateWithPath: nil, bundleID: "com.apple.mobilesafari")!
         loadWebPage("https://www.google.com", waitForLoadToFinish: true)
 
         waitForExistence(app.buttons["HomeView.settingsButton"])
         app.buttons["HomeView.settingsButton"].tap()
-        let safariButton = app.collectionViews.cells.element(boundBy: 5)
+        let safariButton = app.collectionViews.buttons["Open in Default Browser"]
```

Comment:
The order of menu item has been changed. I guess this step is to "Open in Default Browser". `5` is "Find in Page".